app/javascript/controllers/apex_dropdays_controller.js

```diff
@@ -0,0 +1,58 @@
+import { Controller } from "@hotwired/stimulus"
+import { buildNumericChart, LINE_TYPE } from "util/apex"
+
+export default class extends Controller {
+  static values = {
+    id: String,
+    seriesData: Array,
+    range: Number,
+    interval: String,
+  }
+
+  static targets = ["chart"]
+
+  connect() {
+    const series = this.buildSeries()
+    const title = `${this.intervalValue.toLowerCase()}s since episode drop`
```

Comment:
Yeah, lots of stuff that could be i18n'd at some point, and passed in as stimulus values.  Seems like a good separate-PR task though.  This all works to get started.